server/apiserver/argoserver.go

```diff
@@ -56,10 +58,14 @@ import (
 	"github.com/argoproj/argo-workflows/v3/workflow/hydrator"
 )
 
-const (
-	// MaxGRPCMessageSize contains max grpc message size
-	MaxGRPCMessageSize = 100 * 1024 * 1024
-)
+func getGRPCMessageSize() int {
```

Comment:
I'd just use `func init()` to se the variable - panic in invalid value

Reply:
Oh, ok. I was not familiar with init() fun capabilities, which are very nice and useful.
I will fix and update the PR.
Thanks! David

Reply:
Hi @alexec,

I updated the PR as requested. Hope now it is better.
Thank you! David